### PR TITLE
Fix Storybook - jest global usage within stubs

### PR DESCRIPTION
**Pull request by lb-**
* Do not use jest inside stubs.js to prevent Storybook from crashing
* Based on a change in #10865 - just adding an Eslint warning so we do not create this issue again

### CODE REVIEW

**Comment by lb-** [comment]
@laymonage - do you mind taking a look, maybe either include when you merge in or it should be ok to merge in after (hopefully Git does not complain).

File: client/tests/stubs.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
+/* eslint no-restricted-globals: ["error", { "name": "jest", "message": "jest is not available in Storybook." }] */
+
 /**
- * Test stubs to mirror available global variables.
+ * Test stubs to mirror available global variables in Jest tests
+ * and Storybook, avoid using the jest global as this is not
+ * availalbe in Storybook.
```
Comment (laymonage) [inline client/tests/stubs.js:6]:
```suggestion
 * available in Storybook.
```

**Review by laymonage — APPROVED** [review]
I should be able to just drop the similar commit from my PR as I rebase. Love the added ESLint rule, thanks!

File: client/tests/stubs.js (hunk 1 above)
Comment (lb-) [reply client/tests/stubs.js:6]: Argh. Thanks. And yeah. Happy for this go just go in the code however works

**Pull request merged**